notebooks/SMAP/2.0 Read and Plot SMAP data.ipynb

```diff
@@ -0,0 +1,549 @@
+{
```

Comment:
<p>This is a "bandaid" for insufficient information in the SMAP HDF5 files.  I also do not think it is the correct approach.  The data are projected so I think we need to provide the projected coordinates not the geographic coordinates (latitude and longitude) and show how these can be use to plot the data.  This is important because CF-Conventions no longer require latitude and longitude but files should have projected coordinates and a CRS.</p><p> </p><p>If we use latitude and longitude there should be a url for these datasets.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/nsidc/NSIDC-Data-Tutorials/pull/57/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>